dbflute-engine/src/main/java/org/dbflute/task/bs/DfAbstractDbMetaTexenTask.java

```diff
@@ -33,7 +29,6 @@
     // ===================================================================================
     //                                                                           Attribute
     //                                                                           =========
-    protected final List<FileSet> _filesets = new ArrayList<FileSet>();
```

Comment:
Is this line deleted for unused definition?

Reply:
yes.
also confirmed build-torque.xml.

Reply:
Thanks. I see.